Change python grid calc for MTIA back to python mode

nandesuka opened the pull request:
Differential Revision: D83000165




cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @jerryzh168 @voznesenskym @penguinwu @EikanWang @Guobing-Chen @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

blaine-rister reviewed (approved):
LGTM, but could you please update the PR title to reflect the new version?

nandesuka committed:
Change python grid calc for MTIA back to python mode (#163601)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/163601

Test Plan: CI

Reviewed By: blaine-rister

Differential Revision: D83000165
